lib/bitcoin/versions/latest/NormalizedFeeCalculator.ts

```diff
@@ -1,19 +1,66 @@
+import IBlockMetadataStore from '../../interfaces/IBlockMetadataStore';
 import IFeeCalculator from '../../interfaces/IFeeCalculator';
 
 /**
  * `IFeeCalculator` implementation.
  */
 export default class NormalizedFeeCalculator implements IFeeCalculator {
 
+  constructor (
+    private blockMetadataStore: IBlockMetadataStore,
+    private genesisBlockNumber: number,
+    private initialNormalizedFee: number,
+    private lookBackWindowInterval: number,
+    private fluctuationRate: number) {}
+
   /**
    * Initializes the Bitcoin processor.
    */
   public async initialize () {
     console.log(`Initializing normalized fee calculator.`);
   }
 
-  public getNormalizedFee (_block: number): number {
-    // TODO: Issue #783 - Simplify normalized fee calculation algorithm.
-    return 10;
+  public async getNormalizedFee (block: number): Promise<number> {
+    if (block < this.genesisBlockNumber) {
+      // No normalized fee for blocks that exist before genesis
+      return 0;
```

Comment:
I think it should be handled because the it is a reachable path from the API's standpoint. I can ping bitcoin service manually with that request and break it. I don't think it should be treated as an unreachable path just because Core can't reach it.